src/utils/DatasetsUtils.js

```diff
@@ -84,6 +84,9 @@ const uploadZipWithFetchApi = async (organizationId, workspaceId, datasetId, fil
 
 const isVisibleInDatasetManager = (dataset) => dataset?.additionalData?.webapp?.visible?.datasetManager === true;
 
+// TODO: add check on dataset status if created by an ETL script
+const isVisibleInScenarioCreation = (dataset) => dataset?.additionalData?.webapp?.visible?.scenarioCreation === true;
```

Comment:
This modification seems to be already in the last commit. Is it possible to merge this two modifs in one ?

Reply:
@nborde-CSM  Not sure if you mean the two functions `isVisibleInDatasetManager` and `isVisibleInScenarioCreation`, but these are two different functions, related to distinct features and distinct tickets, hence the two distinct commits 🙂